documentation/modules/auxiliary/scanner/http/gitlab_graphql_user_enum.md

```diff
@@ -0,0 +1,235 @@
+## Vulnerable Application
+
+### Description
+
+This module queries the GitLab GraphQL API without authentication to acquire the list of
+GitLab users (CVE-2021-4191). The module works on all GitLab versions from 13.0 up to
+14.8.2, 14.7.4, and 14.6.5.
+
+Exploitation will result in acquiring a list of valid GitLab usernames.
+
+### Installation
+
+#### GitLab 13.10.2 on Ubuntu 20.04.2 x64
```

Comment:
Docker steps that might be worth copy/pasta'ing/modifying

https://github.com/rapid7/metasploit-framework/blob/2cd6e66791fc8a7c70261ea7ddde9472a30341c2/documentation/modules/exploit/multi/http/gitlab_file_read_rce.md#setup

Reply:
I actually think my `.deb` based instructions are much easier :shrug: Unless they are unacceptable, I prefer to use them in documentation.